roles/variables/vars/main.yml

```diff
@@ -1463,15 +1464,15 @@ ksql_properties:
     properties:
       public.key.path: "{{rbac_enabled_public_pem_path}}"
       confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
+      ksql.security.extension.class: io.confluent.ksql.security.KsqlConfluentSecurityExtension
```

Comment:
why did we move these properties to rbac instead of rbac_ldap ?

Reply:
Because these configs are needed for all auth_modes with rbac.

Reply:
Authentication plugin deals with Authentication
Security extension class deals with Authorization.